app/src/pages/inside/common/stackTraceMessageBlock/stackTraceMessageBlock.jsx

```diff
@@ -96,8 +96,9 @@ export class StackTraceMessageBlock extends Component {
     }
 
     if (!this.state.expanded) {
-      const { onOpenStackTraceEvent } = this.props.eventsInfo;
+      const { onOpenStackTraceEvent, clickExpandEvent } = this.props.eventsInfo;
       onOpenStackTraceEvent && this.props.tracking.trackEvent(onOpenStackTraceEvent());
+      clickExpandEvent && this.props.tracking.trackEvent(clickExpandEvent);
```

Comment:
Looks like `clickExpandEvent` and `onOpenStackTraceEvent` are track the same thing at the same time.
Could we use just one of them?